ui/packages/ui/src/app/pages/connectors/ConnectorListItem.tsx

```diff
@@ -93,8 +94,14 @@ export const ConnectorListItem: React.FunctionComponent<IConnectorListItemProps>
                 </div>
               </DataListCell>,
               <DataListCell key={3} width={1}>
-                <div>
-                  Tasks: {getTaskStateStr(props.taskStates)}
+                <div className="tasks-list">
+                {
+                  getTaskStateStr(props.taskStates).map((task, index) => {
```

Comment:
if getTaskState returns the map, just pass the map key to <ConnectorTasks> instead of index.